packages/plotly-demo/src/scatterplot.js

```diff
@@ -55,15 +55,22 @@ function makePlotly(x, y) {
   return Plotly.newPlot("vis", traces, layout, config);
 }
 
-const getAhoiConfig = () => {
+const getAhoiConfig = () => {  
   const defaultOnboardingMessages = generateBasicAnnotations(EVisualizationType.SCATTERPLOT, chart);
   const extendedOnboardingMessages = defaultOnboardingMessages.map((d) => ({
     ...d,
-    text: "test123"
```

Comment:
please add the marker object to the extendedOnboardingMessages here and then pass the extendedOnboardingMessages below in the ahoiConfig, instead of mapping over the defaultOnboardingMessages.